Dockerfile

```
@@ -0,0 +1,14 @@
+# Build the ray binary
+FROM golang:1.12.5 as builder
```

```suggestion
FROM golang:1.12.5-alpine as builder
```

Since we are using alpine now, then I suggest use the alpine in builder, too.